src/test/resources/uk/gov/ons/ctp/response/action/action.feature

```diff
@@ -363,12 +368,12 @@ Feature: Validating action requests
     And the response should contain the field "updatedDateTime"
 
   # 400
-  Scenario: Put request to actionservice for specified actionId invalid json
-    Given I make the PUT call to the actionservice actions endpoint by actionId with invalid input "e71002ac-3575-47eb-b87f-cd9db92bf101"
-    When the response status should be 400
-    And the response should contain the field "error.code" with value "VALIDATION_FAILED"
-    And the response should contain the field "error.message" with value "Provided json is incorrect."
-    And the response should contain the field "error.timestamp"
+#  Scenario: Put request to actionservice for specified actionId invalid json

```

Comment:
Is it worth removing this and the associated step altogether?

Reply:
I think we need to go through the commented out tests at a later date and sort through the relevant ones. For now I'm happy to leave this commented out as it was working previously.